cli/cmd/cluster-apply.go

```diff
@@ -88,6 +89,12 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 		ctxLogger.Fatalf("Verify cluster: %v", err)
 	}
 
+	// Update all the pre installed namespaces with lokomotive specific label.
+	// `lokomotive.kinvolk.io/name: <namespace_name>`.
+	if err := updateInstalledNamespaces(kubeconfig); err != nil {
```

Comment:
If we do that also on AKS, does it retain the labels?

Reply:
yes, I checked for kube-system. It does retain labels.

Reply:
Ok, thanks!